### PR TITLE
Fixes pugme by hardcoding wikimedia commons pug images

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

File: scripts/pugme.coffee [hunk 1]
```diff
@@ -12,17 +12,25 @@
...
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Mops_555.jpg/418px-Mops_555.jpg',
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Pug_wearing_a_hat.JPG/401px-Pug_wearing_a_hat.JPG',
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/Sad-pug.jpg/640px-Sad-pug.jpg',
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/Pug_dog_nose_face_detail.JPG/640px-Pug_dog_nose_face_detail.JPG',
+    'https://upload.wikimedia.org/wikipedia/commons/4/48/Mopshond2_03-10-2005.jpg',
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/d/d2/Bonny_Bonita.JPG/640px-Bonny_Bonita.JPG'
+  ]
 
   robot.respond /pug me/i, (msg) ->
     msg.http("http://pugme.herokuapp.com/random")
```
Comment (aripollak) [inline scripts/pugme.coffee]: lols, remove the unnecessary HTTP requests now?

**Comment by aripollak** [comment]
Alternatively: https://github.com/hubot-scripts/hubot-pugme/pull/8/files

**Comment by sethwoodworth** [comment]
@aripollak updated

File: scripts/pugme.coffee [hunk 2]
```diff
@@ -12,20 +12,22 @@
...
+  ]
 
   robot.respond /pug me/i, (msg) ->
-    msg.http("http://pugme.herokuapp.com/random")
-      .get() (err, res, body) ->
-        msg.send JSON.parse(body).pug
+    msg.send pugArray[Math.floor(Math.random()*pugArray.length)]
 
   robot.respond /pug bomb( (\d+))?/i, (msg) ->
     count = msg.match[2] || 5
```
Comment (aripollak) [inline scripts/pugme.coffee]: Want to reduce the default to something like 2 now?

Comment (sethwoodworth) [inline scripts/pugme.coffee]: Lets do 3, as pug bomb is used to attempt to clear a screen

**Pull request merged**